config/settings.py

```diff
@@ -86,12 +89,12 @@
 # Database
 # https://docs.djangoproject.com/en/4.1/ref/settings/#databases
 
-# TODO: load database from config, consider dj-database-url package
 DATABASES = {
-    "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
-    }
+    'default': dj_database_url.config(
```

Comment:
What if I want to test the code before pushing to heroku? Don't I need sqlite3 for the development environment?

Reply:
I would mirror dev environment as much as production, so I suggest moving to postgres locally as well (and set `DATABASE_URL` in your env, it will be something like this: `DATABASE_URL=postgres://postgres:password@0.0.0.0:5432/your-db`)

Reply:
Or should I install Postgres? Before I used sqlite3 before creating this branch

Reply:
This is what I am doing now:

```
DATABASES = {
    'default': dj_database_url.config(
        default='sqlite:///' + str(BASE_DIR / "db.sqlite3"),
    )
}
```

![image](https://github.com/vnoelifant/movie-search-app/assets/43191157/19b09abc-a3df-4d0f-95f2-bcb4c20a88bd)

Reply:
sqlite is fine for starters, but it won't work in most cloud based solutions (including Heroku) because it's file-based

so you'll need postgres and as per best practice it's then also best to mirror your local setup, so yes, please install postgres and then update your .env to point to the db url as commented above, then change settings.py to this:

```
DATABASES = {
    'default': dj_database_url.config(
        default=config('DATABASE_URL')
    )
}
```

Reply:
Got it, will do, thanks!